log_http.go

```diff
@@ -37,8 +37,18 @@ func (h *HTTPLogger) Start(p *Processor) error {
 	h.processor = p
 
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		host, port, _ := net.SplitHostPort(r.RemoteAddr)
-		ck := NewConnKeyByString(host, port)
+		host, port, err := net.SplitHostPort(r.RemoteAddr)
+		if err != nil {
+			logger.Error(err)
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		ck, err := NewConnKeyByString(host, port)
+		if err != nil {
+			logger.Error(err)
+			http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
would a honeypot want to return a freki-specific error back to the network scanner?

Reply:
I was contemplating this as well, but considering that the http logger can be considered a reference implementation, I thought it makes sense to handle the error visibly for someone getting familiar with Freki.